src/styl/responsive/mobile.styl

```diff
@@ -176,19 +188,19 @@ $filters-height = 29px
 #support-form
   flex-wrap: nowrap
 
-::-webkit-input-placeholder // Chrome/Opera/Safari
+::-webkit-input-placeholder // @stylint ignore
```

Comment:
what is all this about? is this not just generatable with a mixin of some sort?

Reply:
this is stylint calling this prop invalid. you can add it to the valid props, or you can just ignore it like this. was very tired of seeing the error. if there’s a mixin then go ahead